internal/receiver/scriptedinputsreceiver/script_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Splunk, Inc.
```

Comment:
This should be in `tests/receivers/scriptedinputsreceiver` and still in the `tests` package like all the rest are.

Reply:
When I run this in the correct location it appears to not be functional in the container:

```
2023-09-13T13:57:03.219Z	error	scriptedinputsreceiver/commander.go:84	Error in cmd wait: %v	{"kind": "receiver", "name": "scripted_inputs/cpu", "data_type": "logs", "error": "exit status 1"}
github.com/signalfx/splunk-otel-collector/internal/receiver/scriptedinputsreceiver.(*commander).watch
	github.com/signalfx/splunk-otel-collector/internal/receiver/scriptedinputsreceiver/commander.go:84
2023-09-13T13:57:13.215Z	warn	scriptedinputsreceiver/operator.go:108	Script didn't complete within configured interval.{script_name 15 0 cpu <nil>}	{"kind": "receiver", "name": "scripted_inputs/cpu", "data_type": "logs"}
```

I think it'd be worth improving stderr/startup failure behavior and resolving this issue before including as a component. I also wonder if all the other supported scripts should be added first as well.

Reply:
When I placed the test into proper directory I got the same result, what would be your suggestion on how to diagnose the failure in the container (locally on my MAC it works perfectly)

Reply:
initially addressed by container restriction but stderr handling should be added in later PR imo.